clean_md_commodities.py

```diff
@@ -22,287 +22,298 @@
 not_in = lower_md_comm.difference(lower_gen_comm)
 intersection = lower_md_comm.intersection(lower_gen_comm)
 
-df=pd.DataFrame(md_files)
-df.replace(to_replace ='BAIRITE', value ="BARYTE") 
+#These codes run
+
+df1=pd.DataFrame(md_files)
+
+df1=df1.replace(to_replace ='BAIRITE', value ="BARYTE", regex= True)
+
+df2=pd.DataFrame(lower_gen_comm)

```

Comment:
Not sure why you are making replacements on this array - this is just a lower case array version of the commodities dataframe. It was created in order to be able to do an intersection on the commodities in the mining district files and the master list - but if you only replace in `lower_gen_comm`, you'll have to figure out how to get those replacements back into the commodities dataframe before you write that to an excel spreadsheet.